Sources/MapboxNavigation/NavigationComponent.swift

```diff
@@ -15,6 +16,11 @@ public protocol NavigationMapInteractionObserver: AnyObject {
      Called when the NavigationMapView centers on a location.
      */
     func navigationViewController(didCenterOn location: CLLocation)
+    
+    /**
+     Called when the NavigationMapView updates on style.
+     */
+    func navigationViewController(updateTo styleURI: StyleURI?)
```

Comment:
Propose rename:

```suggestion
    func navigationViewController(willUpdateTo styleURI: StyleURI?)
```

Reply:
Thanks for the suggestion, I'll update the name.